### PR TITLE
RELATED: BB-1112 Arithmetic measure object was modified when it was converted into the AFM

### PR DESCRIPTION
**Pull request by xMort**

### CODE REVIEW

File: src/DataLayer/converters/MeasureConverter.ts [hunk 1]
```diff
@@ -156,7 +157,7 @@ function convertArithmeticMeasureDefinition(definition: IArithmeticMeasureDefini
     const { arithmeticMeasure } = definition;
     return {
         arithmeticMeasure: {
-            measureIdentifiers: arithmeticMeasure.measureIdentifiers.splice(0),
+            measureIdentifiers: cloneDeep(arithmeticMeasure.measureIdentifiers),
```
Comment (martinmosko) [inline src/DataLayer/converters/MeasureConverter.ts]: is this only one solution how to fix it? 

using cloneDeep looks to me like covering another issue... :/

Comment (xMort) [reply src/DataLayer/converters/MeasureConverter.ts]: I need to create a new array with a copy of the elements in it. What other approaches would you suggest?

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-js-opensource-repository-check/102/ : SUCCESS in 7s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-js-unit-tests-zuul-docker-el7/119/ : SUCCESS in 1m 19s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-js-opensource-repository-check/104/ : SUCCESS in 6s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-js-unit-tests-zuul-docker-el7/121/ : SUCCESS in 35s

**Comment by gdgate** [comment]
Build failed (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-js-merge-compliance-check/36/ : FAILURE in 12s
- gooddata-js-unit-tests-zuul-docker-el7 : SKIPPED

**Comment by tomassatinsky** [comment]
LGTM

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-js-merge-compliance-check/37/ : SUCCESS in 13s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-js-unit-tests-zuul-docker-el7/122/ : SUCCESS in 37s

**Pull request merged**